Cards/Effects/EffectBase.gd

```diff
@@ -13,4 +13,20 @@ func build_target_list(target: Entity) -> Array[Entity]:
 
 # To be called by child classes through apply_effect. But since there might be multiple effects that need it, it's available in the parent class
 func _apply_status(status: StatusBase, caster: Entity, target: Entity) -> void:
-    target.get_status_component().add_status(status, caster)
\ No newline at end of file
+    target.get_status_component().add_status(status, caster)
+
+func calculate_value(name: String, caster: Entity, target: Entity, value: int) -> int:
```

Comment:
using a String for "draw" and "damage" feels a bit too loose. Maybe adding an enum of like `CardType` maybe better for readability?

Reply:
I totally agree, i was thinking the same when i was writing it.

The reason for this is that in my dictionary of possible modifiers, it's (String, Data), so I need to use strings (and an enum is int). So i'm not exactly sure, I guess I could maybe change the possible modifiers to be from an enum instead, this'll probably be the best.

Reply:
I think the comments of this review can be solved